sw/device/lib/testing/keymgr_testutils.h

```diff
@@ -0,0 +1,22 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_TESTING_KEYMGR_TESTUTILS_H_
+#define OPENTITAN_SW_DEVICE_LIB_TESTING_KEYMGR_TESTUTILS_H_
+
+#include "sw/device/lib/dif/dif_keymgr.h"
+
+/**
+ * Issue an advance operation and wait until the operation is completed
+ */
+void keymgr_advance_state(const dif_keymgr_t *keymgr,
```

Comment:
```suggestion
void keymgr_testutils_advance_state(const dif_keymgr_t *keymgr,
```

Likewise below. The testiutils follow a specific naming convention.